lib/presentation/crowdaction/crowdaction_comments/parts/comment_item.dart

```diff
@@ -0,0 +1,53 @@
+part of '../crowdaction_comments_page.dart';
+
+/// The comment body that includes the commenter's avatar and name
+/// along with the comment content.
+///
+/// This also has [CommentActions] at the bottom.
+///
+/// Currently, this takes a [CrowdactionCommentDto] variable [comment],
```

Comment:
DTO's shouldn't be used in presentation, DTO = Data Transfer Objects, it's the middleman between the API and our domain.